jumeg_math.py

```diff
@@ -81,4 +79,44 @@ def calc_tkeo(signal):
     tkeo[-1] = 0.
 
     # return results
-    return tkeo
\ No newline at end of file
+    return tkeo
+
+
+##################################################
+#
+# destroy phase/time info on any data array
+#
+##################################################
+def shuffle_data (data_trials, mode='shuffle'):
```

Comment:
Okay, once you push it, I'll also test with the jumeg_test script and we can merge.